src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        for (int i=1; i <= 5; i++) {
+            System.out.println(figureSupplier.getRandomFigure().draw());
+        }
```

Comment:
по условию тебе нужно половину массива заполнить случайными фигурами, а половину рандомными
```suggestion
        Figure[] figures = new Figure[6];

        for (int i = 0; i < figures.length / 2; i++) {
            figures[i] = figureSuplier.getRandomFigure();
            figures[i].draw();
        }

        for (int i = figures.length / 2; i < figures.length; i++) {
            figures[i] = figureSuplier.getDafaultFigure();
            figures[i].draw();
        }
```